### PR TITLE
prv candidates using oid as main identifier of alerts

### PR DESCRIPTION
**Pull request by AlxEnashi**
## Summary of the changes

Oid is the main key for alerts, detections and non detections

## If the change is BREAKING, please give more details about the breaking changes

breaks pipeline if other steps are not updated

## About this PR:

- [x] You added the necessary documentation for the team to understand your work.
- [x] You linked this PR to the corresponding issue or user story.
- [x] Your changes fulfill the Definition of Done of the issue or user story.
- [x] Your changes were tested with manual testing before being submitted.
- [ ] Your changes were tested with automatic unit tests.
- [ ] Your changes were tested with automatic integration tests.

### CODE REVIEW

File: prv_candidates_step/prv_candidates_step/core/strategy/ztf.py [hunk 1]
```diff
@@ -65,6 +65,7 @@ def parse_message(cls, candidate: dict, alert) -> dict:
         model.update(
             {
                 "aid": acopy["aid"],
+                "oid": acopy["oid"],
```
Comment (dirodriguezm) [inline prv_candidates_step/prv_candidates_step/core/strategy/ztf.py:68]: Esto no viene ya en `generic` ?

File: prv_candidates_step/prv_candidates_step/core/strategy/ztf.py [hunk 2]
```diff
@@ -128,6 +129,7 @@ def parse_message(cls, forced_photometry: dict, alert: dict) -> dict:
...
```
Comment (dirodriguezm) [inline prv_candidates_step/prv_candidates_step/core/strategy/ztf.py:132]: Lo mismo que el comentario anterior

File: prv_candidates_step/prv_candidates_step/core/strategy/ztf.py [hunk 3]
```diff
@@ -165,6 +167,7 @@ def parse_message(cls, candidate: dict, alert: dict) -> dict:
         model = cls._Model(**generic, stamps=stamps, extra_fields=extra_fields)
         model = model.to_dict()
         model.update({"aid": acopy["aid"]})
+        model.update({"oid": acopy["oid"]})
```
Comment (dirodriguezm) [inline prv_candidates_step/prv_candidates_step/core/strategy/ztf.py:170]: lo mismo que el comentario anterior

File: prv_candidates_step/tests/integration/test_step.py [hunk 4]
```diff
@@ -62,7 +62,7 @@ def assert_scribe_has_non_detections(message):
     data = json.loads(message["payload"])
     assert data["collection"] == "non_detection"
     assert data["type"] == "update"
-    assert "aid" in data["criteria"]
+    #assert "aid" in data["criteria"]
```
Comment (dirodriguezm) [inline prv_candidates_step/tests/integration/test_step.py:65]: En lugar de comentar mejor borrar ?

**Review by dirodriguezm — CHANGES_REQUESTED** [review]

File: prv_candidates_step/prv_candidates_step/core/strategy/ztf.py (hunk 1 above)
Comment (AlxEnashi) [reply prv_candidates_step/prv_candidates_step/core/strategy/ztf.py:68]: nop

File: prv_candidates_step/tests/integration/test_step.py (hunk 4 above)
Comment (AlxEnashi) [reply prv_candidates_step/tests/integration/test_step.py:65]: Ah se me fue borrarlo, lo comente para ver si ese era el error o era otro... y si era ese

File: prv_candidates_step/prv_candidates_step/core/strategy/ztf.py (hunk 3 above)
Comment (AlxEnashi) [reply prv_candidates_step/prv_candidates_step/core/strategy/ztf.py:170]: nop, no viene

**Review by dirodriguezm — APPROVED** [review]

**Pull request merged**